### PR TITLE
merge albuminfo & songinfo xml files

### PR DESCRIPTION
**Pull request by ronie**
cleanup and merge of DialogAlbumInfo.xml & DialogSongInfo.xml

these controls have been removed:
- "3" image thumb (don't think skins are using this)
- "4" textbox (skins can add their own textboxes with whatever info they want) 
- "5" button to toggle between textbox and list (no longer needed)

new xml name: DialogMusicInfo.xml
core will now handle the visibility and set the labels of the required buttons.

(changes made by @HitcherUK in https://github.com/xbmc/xbmc/pull/8015 are included)

### CODE REVIEW

File: addons/skin.confluence/720p/includes.xml [hunk 1]
```diff
@@ -1407,4 +1423,226 @@
...
+			<visible>!IsEmpty(ListItem.Property(Role.Composer))</visible>
+		</item>
+		<item>
+			<label>$LOCALIZE[29905]:</label>
+			<label2>$INFO[ListItem.Property(Role.Conductor)]</label2>
+			<onclick>noop</onclick>
+			<visible>!IsEmpty(ListItem.Property(Role.Conductor))</visible>
+		</item>
+		<item>
+			<label>Orchestra:</label>
```
Comment (HitcherUK) [inline addons/skin.confluence/720p/includes.xml]: Needs localising. See updated DialogSongInfo - https://github.com/DaveTBlake/xbmc/blob/ArtistRoles/addons/skin.confluence/720p/DialogSongInfo.xml

File: addons/skin.confluence/720p/includes.xml [hunk 2]
```diff
@@ -1407,4 +1423,226 @@
...
+			<label2>$INFO[ListItem.Property(Role.Orchestra)]</label2>
+			<onclick>noop</onclick>
+			<visible>!IsEmpty(ListItem.Property(Role.Orchestra))</visible>
+		</item>
+		<item>
+			<label>Lyricist:</label>
```
Comment (HitcherUK) [inline addons/skin.confluence/720p/includes.xml]: As above

File: addons/skin.confluence/720p/includes.xml [hunk 3]
```diff
@@ -1407,4 +1423,226 @@
...
+			<label2>$INFO[ListItem.Property(Role.Lyricist)]</label2>
+			<onclick>noop</onclick>
+			<visible>!IsEmpty(ListItem.Property(Role.Lyricist))</visible>
+		</item>
+		<item>
+			<label>Remixer:</label>
```
Comment (HitcherUK) [inline addons/skin.confluence/720p/includes.xml]: As above

File: addons/skin.confluence/720p/includes.xml [hunk 4]
```diff
@@ -1407,4 +1423,226 @@
...
+			<label2>$INFO[ListItem.Property(Role.Remixer)]</label2>
+			<onclick>noop</onclick>
+			<visible>!IsEmpty(ListItem.Property(Role.Remixer))</visible>
+		</item>
+		<item>
+			<label>Arranger:</label>
```
Comment (HitcherUK) [inline addons/skin.confluence/720p/includes.xml]: As above

File: addons/skin.confluence/720p/includes.xml [hunk 5]
```diff
@@ -1407,4 +1423,226 @@
...
+			<label2>$INFO[ListItem.Property(Role.Arranger)]</label2>
+			<onclick>noop</onclick>
+			<visible>!IsEmpty(ListItem.Property(Role.Arranger))</visible>
+		</item>
+		<item>
+			<label>Engineer:</label>
```
Comment (HitcherUK) [inline addons/skin.confluence/720p/includes.xml]: As above

File: addons/skin.confluence/720p/includes.xml [hunk 6]
```diff
@@ -1407,4 +1423,226 @@
...
+			<label2>$INFO[ListItem.Property(Role.Engineer)]</label2>
+			<onclick>noop</onclick>
+			<visible>!IsEmpty(ListItem.Property(Role.Engineer))</visible>
+		</item>
+		<item>
+			<label>Producer:</label>
```
Comment (HitcherUK) [inline addons/skin.confluence/720p/includes.xml]: As above

File: addons/skin.confluence/720p/includes.xml [hunk 7]
```diff
@@ -1407,4 +1423,226 @@
...
+			<label2>$INFO[ListItem.Property(Role.Producer)]</label2>
+			<onclick>noop</onclick>
+			<visible>!IsEmpty(ListItem.Property(Role.Producer))</visible>
+		</item>
+		<item>
+			<label>DJMixer:</label>
```
Comment (HitcherUK) [inline addons/skin.confluence/720p/includes.xml]: As above

File: addons/skin.confluence/720p/includes.xml [hunk 8]
```diff
@@ -1407,4 +1423,226 @@
...
+			<label2>$INFO[ListItem.Property(Role.DJMixer)]</label2>
+			<onclick>noop</onclick>
+			<visible>!IsEmpty(ListItem.Property(Role.DJMixer))</visible>
+		</item>
+		<item>
+			<label>Mixer:</label>
```
Comment (HitcherUK) [inline addons/skin.confluence/720p/includes.xml]: As above

**Comment by HitcherUK** [comment]
Thanks once again ronie.

**Comment by ronie** [comment]
thx @HitcherUK, i hadn't noticed these weren't localized :-)

**Comment by DaveTBlake** [comment]
Thanks both for your help getting additional song info visible.
For localization the extra values are in the merged verion of strings.po now.

**Comment by DaveTBlake** [comment]
Could mood be added to the list of items for MusicInfoSongInfo too, below genre possibly. 
Songs have had the mood tag processed for ages, you can search on it etc. but never see the value, obviously an oversight. Info lable is Listitem.Mood.

**Comment by ronie** [comment]
rebased, localized and added mood infolabel.

**Comment by MartijnKaijser** [comment]
@mkortstiege for a code check :)

**Comment by mkortstiege** [comment]
Looks good to me.

**Comment by MartijnKaijser** [comment]
jenkins build this please

**Comment by xhaggi** [comment]
+1

**Pull request merged**